ocamldoc/odoc_sig.ml

```diff
@@ -970,6 +973,7 @@ module Analyser =
               match types_ext.ext_args with
               | Cstr_tuple l -> Cstr_tuple (List.map (Odoc_env.subst_type env) l)
               | Cstr_record l ->
+                  let l = Nonempty_list.to_list l in
                   let docs = Record.(doc types) pos_end l in
                   Cstr_record (List.map (get_field env docs) l)
             in
```

Comment:
I would expect this to not typecheck

Reply:
I believe they are two different types, one defined in `Odoc_info`